app/policies/account_policy.rb

```diff
@@ -0,0 +1,63 @@
+class AccountPolicy < TeamDependantPolicy
+  def show?
```

Comment:
can't we move this methods also do TeamDepPol ? probably mostly the same for group, account, item

Reply:
Done

Reply:
Done